wordpress-mobile/WordPress-Android
# Point to GB mobile with JitPack workaround

With a [recent change in GB mobile](https://github.com/wordpress-mobile/gutenberg-mobile/pull/465) and a JitPack issue, the binary builds for gutenberg-mobile (and react-native-aztec) were broken. This PR uses a workaround until the issue is fixed.

To test:

Build and open a post in GB mobile. It should work as normal. Also, Travis/CircleCI should be green on this.

Update release notes:
